packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -0,0 +1,84 @@
+import fs from 'fs-extra';
+import { join, dirname, relative } from 'path';
+import glob from './fs/glob';
+import { FILES_SYMBOL, Lambda } from './lambda';
+import type FileBlob from './file-blob';
+import type { BuildOptions, Files } from './types';
+
+const OUTPUT_DIR = '.output';
+
+/**
+ *
+ * @param buildRuntime a legacy build() function from a Runtime
+ * @param ext the file extension, for example `.py`
+ */
+export async function convertRuntimeToPlugin(
+  buildRuntime: (options: BuildOptions) => Promise<{ output: Lambda }>,
+  ext: string
+) {
+  return async function build({ workPath }: { workPath: string }) {
+    const opts = { cwd: workPath };
+    const files = await glob('**', opts);
+    const entrypoints = await glob(`api/**/*${ext}`, opts);
+    const dir = join(workPath, OUTPUT_DIR, 'inputs', 'runtime-temp');
+    await fs.ensureDir(dir);
+    for (const entrypoint of Object.keys(entrypoints)) {
+      const { output } = await buildRuntime({
+        files,
+        entrypoint,
+        workPath,
+        config: {}, // TODO: what should config be?
+        meta: {}, // TODO: what should meta be?
```

Comment:
Not sure about `meta`, but `config` should probably be what we use for `zero-config`.

Reply:
Added in 5b9cc08